README.md

```diff
@@ -65,15 +79,6 @@ salesy way; the loyalty program is our growth strategy."""
 
 ### Environment Variables
 
-Look at the [`.env.sample`](.env.sample) file to see the environment variables needed to run the project.
-
-There are other configurations not detailed in `.env.sample`, we recommend you to [read our documentation](https://dialog.talkd.ai/settings#environment-variables) that discusses configuration.
-
-## Run the project
+Look at the [`.env.sample`](.env.sample) file to see the environment variables needed to run the project. Make sure to add the OpenAI token and paths to the `.csv` and `.toml` files. We recommend you to [read our documentation](https://dialog.talkd.ai/settings#environment-variables) that discusses configuration.
```

Comment:
The csv and toml sentence is confusing, we need to make it clear that the user should only put the tokens and variables inside the toml/env-vars.